package.json

```
@@ -33,5 +33,5 @@
     "globals": "^16.0.0",
     "markdown-link-check": "3.12.2",
     "prettier": "3.5.3"
-  }
+  },
```

```suggestion
  }
```

I missed removing the comma